src/JsSse.php

```diff
@@ -190,8 +190,10 @@ protected function initSse(): void
         }
 
         // prevent buffering
-        if (ob_get_level()) {
+        $levels = ob_get_level();
+        for ($i = 0; $i < $levels; $i++) {
             ob_end_flush();
         }
+        flush();
```

Comment:
please explain why `flush()` is helpful

Reply:
> please explain why `flush()` is helpful
I don't know. The code is taken from the example I gave. If you think it is not needed, delete it.